contracts/base/Escrow/Dapps/Uniswap.sol

```diff
@@ -18,75 +24,99 @@ import "./IDApp.sol";
 /**  Visit https://docs.openzeppelin.com/upgrades/2.6/proxies#upgrading-via-the-proxy-pattern for   **/
 /**  more information.                                                                              **/
 /*****************************************************************************************************/
-contract Uniswap is IDApp {
+/**
+    @notice This contract is used to define Uniswap dApp actions available. All dapp actions are invoked via 
+        delegatecalls from Escrow contract, so this contract's state is really Escrow.
+    @author develop@teller.finance
+ */
+contract Uniswap is IUniswap {
     using AddressLib for address;
+    using Address for address;
+
+    /* State Variables */
+    // State is shared with Escrow contract as it uses delegateCall() to interact with this contract.
+    
+     function() external payable {}
```

Comment:
Dapp contracts should not have a fall back function. More importantly, they should not be able to hold ANY funds.

Reply:
Payable fallback function added on Mock for tests only.